Gemfile

```diff
@@ -24,7 +24,9 @@ gem 'dough-ruby',
 gem 'jquery-rails'
 gem 'kaminari'
 gem 'letter_opener', group: :development
-gem 'mas-rad_core', '0.0.58'
+gem 'mas-rad_core',
+    github: 'moneyadviceservice/mas-rad_core',
+    branch: 'dev/add_website_address_to_principal_factory'
```

Comment:
[There was but it's merged.](https://github.com/moneyadviceservice/mas-rad_core/pull/73). I'll do the rubygems wrangling last thing before I merge this (in case feedback means touching that repo again)